src/main/java/com/squareup/gifencoder/LzwEncoder.java

```diff
@@ -44,8 +44,11 @@
   private int codeSize;
   private List<Integer> indexBuffer = new ArrayList<>();
 
+  /**
+   * @param numColors Padded number of colors, must be a power of 2
```

Comment:
Should we validate this and throw IllegalArgumentException otherwise?

Reply:
Will do in a a followup PR.